src/ReachSets/ContinuousPost/BFFPS19/BFFPS19.jl

```diff
@@ -0,0 +1,274 @@
+        v = 𝑂[:lazy_inputs_interval]
+        if v isa Int
+            if v == -1
+                𝑂.specified[:lazy_inputs_interval] = lazy_inputs_interval_never
+            elseif v == 0
+                𝑂.specified[:lazy_inputs_interval] = lazy_inputs_interval_always
+            else
+                𝑂.specified[:lazy_inputs_interval] = (k -> k % v == 0)
+            end
+        end
+    end
+
+    # :block_options options: use convenience option for '_init' and '_iter'
+    if !haskey_specified(𝑂, :block_options_init) &&
+            haskey_specified(𝑂, :block_options)
+        𝑂.specified[:block_options_init] = 𝑂[:block_options]
+    end
+    if !haskey_specified(𝑂, :block_options_iter) &&
+            haskey_specified(𝑂, :block_options)
+        𝑂.specified[:block_options_iter] = 𝑂[:block_options]
+    end
+
+    nothing
+end
+
+function validation_BFFPS19(𝑂)
+    # block_options
+    for b_options in [:block_options, :block_options_init, :block_options_iter]
+        if haskey_specified(𝑂, b_options)
+            bo = 𝑂[b_options]
+            if bo isa Type{<:LazySet} || bo isa Type{<:AbstractDirections}
+                # uniform options
+            elseif bo isa Symbol
+                # template directions
+                option = get(Utils.template_direction_symbols, bo, nothing)
+                if option == nothing
+                    throw(DomainError(bo, "if the `$b_options` option " *
+                        "is a Symbol, it must be one of " *
+                        "$(keys(Utils.template_direction_symbols))"))
+                end
+                𝑂.specified[b_options] = option
+            elseif bo isa AbstractVector || bo isa Dict{Int, Any}
+                # mapping
+            elseif bo isa Real || bo isa Pair{<:UnionAll, <:Real}
+                ε = bo isa Real ? bo : bo[2]
+                if ε <= 0
+                    throw(DomainError(ε, "the `$b_options` option must be " *
+                                         "positive"))
+                end
+            elseif b_options == :block_options_iter && bo == nothing
+                # no overapproximation
+            else
+                throw(DomainError(bo == nothing ? "nothing" : bo,
+                    "the `$b_options` option does not accept the given input"))
+            end
+        end
+    end
+
+    nothing
+end
+
+"""
+    BFFPS19 <: ContinuousPost
+
+Implementation of the reachability algorithm for purely continuous linear
+time-invariant systems using block decompositons by S. Bogomolov, M. Forets,
+G. Frehse, A. Podelski, C. Schilling and F. Viry [1].
+
+### Fields
+
+- `options` -- an `Options` structure that holds the algorithm-specific options
+
+### Notes
+
+The following options are available:
+
+```julia
+$(print_option_spec(options_BFFPS19()))
+```
+
+### Algorithm
+
+We refer to [1] for technical details.
+
+[1] [Reach Set Approximation through Decomposition with Low-dimensional Sets
```

Comment:
Wrong citation? This is https://juliareach.github.io/Reachability.jl/latest/publications.html#Reachability-analysis-of-linear-hybrid-systems-via-block-decomposition-1